anyway/utilities.py

```diff
@@ -197,17 +197,28 @@ def delete_all_rows_from_table(conn, table):
     logging.info("Deleting all rows from table " + table_name)
     conn.execute("DELETE FROM " + table_name)
 
-
-def split_query_to_chunks_by_column(base_select, column_to_chunk_by, chunk_size, conn):
-    column_values = fetch_first_and_every_nth_value_for_column(conn, column_to_chunk_by, chunk_size)
-    logging.debug("after fetching every nth column")
-    for index in range(len(column_values)):
-        select = base_select.where(column_to_chunk_by >= column_values[index])
-        if index + 1 < len(column_values):
-            select = select.where(column_to_chunk_by < column_values[index + 1])
-        chunk = conn.execute(select).fetchall()
+def _yield_select_in_chunks(base_select: str, column_to_chunk_by: str, val: typing.Any, next_val: typing.Any, conn, chunk_size: int):
```

Comment:
base_select is not str, as can be seen in 201. I believe it is sqlalchemy.sql.expression.Select
also conn is sqlalchemy.engine.Connection

Reply:
Added 😄